01-start/03-begin.html.md

```diff
@@ -27,12 +27,13 @@ Before we get hacking away on content, we have to create our project and set it
 ``` bash
 mkdir my-new-website
 cd my-new-website
+docpad --offline init
```

Comment:
Not sure why `--offline` is used here... I would omit that. If it is to avoid the initial newsletter signup, or the skeleton asking, perhaps we can inform the user in a better way, rather than just hide those details away from them.